plugins/input/gdal/gdal_featureset.cpp

```diff
@@ -397,6 +397,25 @@ feature_ptr gdal_featureset::get_feature(mapnik::query const& q)
                 feature->set_raster(raster);
                 break;
             }
+            case GDT_Int32:
+            {
+                mapnik::image_gray32s image(im_width, im_height);
+                image.set(std::numeric_limits<std::int32_t>::max());
+                raster_nodata = band->GetNoDataValue(&raster_has_nodata);
+                raster_io_error = band->RasterIO(GF_Read, x_off, y_off, width, height,
+                                                 image.data(), image.width(), image.height(),
+                                                 GDT_Int32, 0, 0, &psExtraArg);
```

Comment:
`psExtraArg` is not declared anywhere /cc @cpaulik

Reply:
Oops. I left that in from https://github.com/mapnik/mapnik/pull/3975. Will take it out

Reply:
:+1: